src/components/IPScan/Status/SequenceList/index.tsx

```diff
@@ -98,6 +108,28 @@ export const IPScanStatus = ({
     }
   }, [job, isImporting]);
 
+  useEffect(() => {
+    const setJobExpiryDate = async (job: MinimalJobMetadata) => {
+      const jobMetadataRequest = await fetch(
+        `https://www.ebi.ac.uk/Tools/services/rest/iprscan5/jobruntime/${job.remoteID}`,
+      );
```

Comment:
Will not work if the job is submitted on the dev/test Job Dispatcher service (`https://wwwdev.ebi.ac.uk/Tools/services/rest/iprscan5`).

Please use the global state so the URL defined in `config.yml` is used, e.g.

https://github.com/ProteinsWebTeam/interpro7-client/blob/21b794ce4bcf3c7f48bcd385bc3507d143ab0324/src/components/IPScan/Search/index.tsx#L375-L385